index.js

```diff
@@ -8,7 +8,8 @@ const process = require('process')
 const TSCONFIG_PATH = process.env.TSCONFIG_PATH
 var upstreamTransformer = null
 try {
-  upstreamTransformer = require('metro-bundler/src/transformer')
+  // handle RN 0.46
+  upstreamTransformer = require('metro-bundler/build/transformer')
```

Comment:
Hi, thanks for the contribution! This line is the only change I need and want to keep from your PR. Can you revert the others please?